skiko/src/iosMain/kotlin/org/jetbrains/skiko/redrawer/MetalRedrawer.ios.kt

```diff
@@ -83,14 +103,15 @@ internal class MetalRedrawer(
 
     private fun draw() {
         // TODO: maybe make flush async as in JVM version.
-        autoreleasepool {
+        autoreleasepool { //todo measure performance without autoreleasepool
             if (!isDisposed) {
                 contextHandler.draw()
             }
         }
     }
 
     fun finishFrame() {
+        removeFrameSubscription()
```

Comment:
Shouldn't we call this (or isPaused = false) in the end of `onDisplayLinkTick` for robustness? I.e. we be sure that every performed frame will be always the only one. In some conditions `finishFrame` can be not called (if `!layer.isShowing()`, for example)

Reply:
Done